apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -103,6 +103,10 @@ export class WorkshopEnrollmentSchoolInfo extends React.Component {
   }
 
   scholarshipInfo(enrollment) {
+    if (enrollment.new_facilitator) {
```

Comment:
This logic, certain facilitators cannot have scholarship, should live in model instead of UI component.